dpc-sdp/ripple-framework
# [R20-1493] Add accessible label to links opening in a new window

**Issue**: https://digital-vic.atlassian.net/browse/R20-1493

### What I did

- Added accessible label in `RplTextLink`
- Added accessible label in `RplDocument`
- Added accessible label in cheerio for `RplContent`
- Updated a unit test to demonstrate
- Added a type abstraction to get rid of `hastitle` and `hassidebar` attrs in dynamic page component rendered markup

### How to test

- Unit test
- e2e: create a link with `target="_blank"` in content managed block on BE, check output on FE

### Checklist




#### For all PR's


#### For new components only
